Atomic/scan.py

```diff
@@ -55,19 +55,36 @@ def get_scan_info(scanner, scan_type):
             raise ValueError(yaml_error)
 
         self.results_dir = os.path.join(self.results, self.args.scanner, self.cur_time)
-        scan_list = self._get_scan_list()
-        for i in scan_list:
-            self.scan_content[i['Id']] = i.get('input')
-
-        # mount all the rootfs
-        self._mount_scan_rootfs(scan_list)
 
+        # Create the input directory
+        if not os.path.exists(self.chroot_dir):
+            os.makedirs(self.chroot_dir)
         if self.debug:
-            util.write_out("Creating the output dir at {}".format(self.results_dir))
+            util.write_out("Created {}".format(self.chroot_dir))
+
+        if self.args.rootfs is None:
+            scan_list = self._get_scan_list()
```

Comment:
I like to think of rootfs as a list.  I would prefer to always treat it as a list.  The problem with having it be sometimes a none and sometimes a list, you end up with code that looks like

if self.args.rootfs:
      for r in self.args.rootfs:
                print r

It it is always a list I don't need the first if.